### ISSUE TITLE
Ctrl+w not deleting word backward in Command Palette

### ISSUE THREAD

**Issue by maptv** [microsoft/vscode#200692]
Type: <b>Bug</b>

Steps to reproduce: Add either of these keybinding: ` { "key": "ctrl+w", "command": "deleteWordLeft" }` or `{ "key": "ctrl+w", "command": "deleteWordLeft", "when": "textInputFocus && !editorReadonly" }` to `keybindings.json`, summon the Command Palette, type some text, press `Ctrl+W`.

Desired result: Delete the previous word in the Command Palette text input.

Acual result: Nothing happens or deletion occurs in the Editor instead of the Command Palette text input.

Notes: 
- `Alt+Backspace` behaves how I want. If I could remap `Ctrl+W` to `Alt+Backspace`, I would be all set.
- This problem persists even if I disable all extensions.

VS Code version: Code 1.85.0 (af28b32d7e553898b2a91af498b1fb666fdebe0c, 2023-12-06T18:17:50.719Z)
OS version: Darwin arm64 23.1.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M2 Max (12 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|32.00GB (0.37GB free)|
|Process Argv|--crash-reporter-id cbd886a3-9f70-4207-ae2e-a09c1b25ee8c|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-neovim|asv|1.3.0
gitlens|eam|14.5.2
html-preview-vscode|geo|0.2.5
codespaces|Git|1.16.4
open-in-vim|jon|0.7.0
black-formatter|ms-|2023.6.0
python|ms-|2023.22.0
vscode-pylance|ms-|2023.11.10
jupyter|ms-|2023.11.1003402403
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
vsliveshare|ms-|1.0.5900
quarto|qua|1.109.0
r|REd|2.8.2
lua|sum|3.7.3
tabnine-vscode|Tab|3.51.0
quokka-vscode|Wal|1.0.602
fig|wit|0.0.7

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
py29gd2263:30899288
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
2e4cg342:30602488
89544117:30613380
vscrp:30673768
showlangstatbar:30737416
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
welcomedialogc:30910334
pythonnosmt12:30797651
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
dsvsc013:30795093
dsvsc014:30804076
dsvsc015:30845448
pythontestfixt:30902429
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementst:30887150
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
aa_t_chat:30882232
dsvsc019cf:30917260

```

</details>

**Comment by maptv** [comment]
Even when I delete all of my settings and keybindings, I cannot bind `Ctrl+W` to delete previous word. If I add the binding to completely vanilla VSCode, pressing `Ctrl+W` deletes the entire line in Command Palette / File Search and brings up a select window popup in the Settings or Keyboard Shortcuts text input. I think there is something in VSCode that prevents this binding from working in any of the text inputs.

**Comment by TylerLeonhardt** [comment]
Yeah I don't think this works because the QuickPick input is not a rich editor like the editor is. It'd be nice to light this up, which we could do if we made the quickpick's editor a monaco instance

**Comment by maptv** [comment]
Is there any way to remap `Ctrl+W` to `Alt+Backspace` in VSCode?
In Vim, I would do something like this: `inoremap <C-w> <A-BS>`.

**Comment by axgn** [comment]
After three years,I'd also like to see this feature implemented,any updates?
